globals.json

```diff
@@ -1191,9 +1191,10 @@
 		"jQuery": false
 	},
 	"yui": {
-		"Y": false,
```

Comment:
Is it safe to remove this?

Reply:
Like I said, it's not a global variable unless the user does this in the global scope:
```
var Y = YUI();